Package.swift

```diff
@@ -1,11 +1,19 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.9
 import PackageDescription
 
 let clibsodiumTarget: Target
 #if os(OSX) || os(macOS) || os(tvOS) || os(watchOS) || os(iOS)
     clibsodiumTarget = .binaryTarget(
         name: "Clibsodium",
         path: "Clibsodium.xcframework")
+#elseif os(Windows)
+    clibsodiumTarget = .target(
+        name: "Clibsodium",
+        path: "Clibsodium-win",
+        publicHeadersPath: "include",
+        linkerSettings: [
+          .linkedLibrary("libsodium"),
```

Comment:
Should this not be `sodium`?  Otherwise, this would be `liblibsodium` I believe.

Reply:
This version works and links `libsodium.lib`

Reply:
Oh, nice!